apps/cowswap-frontend/src/modules/swap/containers/SurplusModalSetup/index.tsx

```diff
@@ -4,33 +4,42 @@ import { useWalletInfo } from '@cowprotocol/wallet'
 
 import { useOrder } from 'legacy/state/orders/hooks'
 
-import { useGetSurplusData } from 'common/hooks/useGetSurplusFiatValue'
 import { CowModal } from 'common/pure/Modal'
+import { OrderProgressBarV2 } from 'common/pure/OrderProgressBarV2'
 import * as styledEl from 'common/pure/TransactionSubmittedContent/styled'
-import { SurplusModal } from 'common/pure/TransactionSubmittedContent/SurplusModal'
 
+import { useTradeConfirmState } from '../../../trade'
 import { useOrderIdForSurplusModal, useRemoveOrderFromSurplusQueue } from '../../state/surplusModal'
+import { useNavigateToNewOrderCallback, useSetupAdditionalProgressBarProps } from '../ConfirmSwapModalSetup'
 
+// TODO: rename?
 export function SurplusModalSetup() {
```

Comment:
Surplus modal is not about just surplus anymore.

In the good to haves to rename/refactor this.